go/kbfs/simplefs/simplefs.go

```diff
@@ -2251,6 +2251,17 @@ func (k *SimpleFS) SimpleFSSetFolderSyncConfig(
 	return err
 }
 
+// SimpleFSClearConflictState implements the SimpleFS interface.
+func (k *SimpleFS) SimpleFSClearConflictState(ctx context.Context,
+	path keybase1.Path) error {
+	// TODO: there's got to be a better way to get a TLF ID
```

Comment:
Yeah you can just copy what's in `getSyncConfig`, or make a helper function for getting the handle:

```go
	t, tlfName, _, _, err := remoteTlfAndPath(path)
	if err != nil {
		return err
	}
	tlfHandle, err := libkbfs.GetHandleFromFolderNameAndType(
		ctx, k.config.KBPKI(), k.config.MDOps(), k.config, tlfName, t)
	if err != nil {
		return err
	}
    tlfID := tlfHandle.TlfID()
```

I guess I was pretty lazy in the function you copied this from.